Code/GraphMol/Chirality.cpp

```diff
@@ -1112,7 +1113,15 @@ void findPotentialStereoBonds(ROMol &mol, bool cleanIt) {
             // ------------------
             // get the CIP ranking of each atom if we need it:
             if (!cipDone) {
-              Chirality::assignAtomCIPRanks(mol, ranks);
+              if (!begAtom->hasProp(common_properties::_CIPRank)) {
+                Chirality::assignAtomCIPRanks(mol, ranks);
+              } else {
+                // no need to recompute if we don't need to recompute. :-)
+                for (unsigned int ai = 0; ai < mol.getNumAtoms(); ++ai) {
```

Comment:
Out of curiosity, what happens if someone makes a copy with AddHs and recalls this function.  Should AddHs remove cip ranks?  Is that a user issue to clear it?

I'll do a build and find out.